includes/class-newspack-popups-model.php

```diff
@@ -69,11 +150,13 @@ public static function retrieve_popup_by_id( $post_id ) {
 	 * Retrieve popup CPT post.
 	 *
 	 * @param WP_Query $query The query to use.
+	 * @param boolean  $include_categories If true, returned objects will include assigned categories.
 	 * @return object Popup object
 	 */
-	protected static function retrieve_popup_with_query( WP_Query $query ) {
-		$popup = null;
+	protected static function retrieve_popup_with_query( WP_Query $query, $include_categories = false ) {
+		$popups = [];
 		while ( $query->have_posts() ) {
+			$popup = null;
```

Comment:
I don't believe this line is needed at all?

Reply:
Removed in https://github.com/Automattic/newspack-popups/pull/34/commits/8276805f65318a9e971857b68e43852817ca60ed